README.md

```diff
@@ -7,61 +7,168 @@
 
 * Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
 * Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+* Create/chain multiple *if-else* statements together to consider additional clauses:
+
+
+
+## Outline / Notes
+
+In the previous lesson we covered Boolean values (namely ````true```` and ````false````) along with the various operations we can perform that result in a Boolean value (namely ````<, ==, <=, >, >=, !=````).
+For example:
+
+````Swift
+3 < 5    // false
+12 >= 12  // true
+let greedoFiredFirst  // true
+! greedoFiredFirst  // false
+````
+
+At this point you may find Booleans interesting but are wondering what they're actually good for.  If that's the case then you're in luck because that's *exactly* what this lesson is about.
 
-```swift
-var age = 19
+So far you've covered computer code that's of the form of directives and declarations; that is, "do this".  For example:
 
-if age < 21 {
-    print("You're not allowed to drink!")
+````Swift
+print("hello")   // hello
+// directive to print "hello"
+
+var x = 5 * 8
+// declaration of variable x
+
+let i = 12 >= 9
+// declaration of constant i
+````
+
+Most programs, however, need an additional component, and that component is Conditionals.  Conditionals are how you tell the program to take different behavior based upon conditions or results.  The simplest form of Conditional is just to do something if an expression is true.  We can express "if class is hard then study more" like this:
+
+````Swift
+if classIsHard {
+	studyMore()
 }
+````
 
-// prints "You're not allowed to drink!"
-```
+We can also specify an "else" clause.  For example, we can express "if class is hard then study more otherwise catch up on sleep" ike this:
 
-* Explain that an *if* statement can provide an alternative set of statements, known as an *else clause*, for situations when the *if* condition is *false*.
+````Swift
+if classIsHard() {
+	studyMore()
+} else {
+	catchUpOnSleep()
+}
```

Comment:
If we run with the format of what I introduced above this comment, I think we can take a similar approach here like this:

---

```swift
func studyMore() {
    print("Study 📖")
}

func catchUpOnSleep() {
    print("Zzzz 💤 😴")
}

var classIsHard = false

if classIsHard {
    studyMore()
} else {   
    catchUpOnSleep()
}

// prints "Zzzz 💤 😴"
```

Similar to above, but we added another function named `catchUpOnSleep()` which prints 'Zzzz 💤 😴' when called on. Then we are creating the `classIsHard` variable but setting it's initial value to `false`. Next we've created an `if-else` statement which should read like a sentence. If the class is hard, then we should study more.. if not, then we should catch up on some sleep. As you can see, what prints to console is 'Zzzz 💤 😴' which means `studyMore()` does not get called, we do not enter those `{ }` curly braces, but we do enter the `{ }` braces after the `else` statement because the first `if` statement evaluated to `false`. If it evaluated to `true`, then 'Study 📖' would have printed to the console and the `catchUpOnSleep()` function would never have gotten called.